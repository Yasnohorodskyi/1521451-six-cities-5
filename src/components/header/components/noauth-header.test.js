
import React from "react";
import renderer from "react-test-renderer";
import {
  MemoryRouter as Router,
} from 'react-router-dom';

import NotAuthHeader from "./noauth-header";

describe(`Render NotAuthHeader`, () => {


  it(`Render NotAuthHeader without store`, () => {

    const NotAuthHeaderTest = renderer.create(
        <Router>
          <NotAuthHeader/>
        </Router>
    ).toJSON();

    expect(NotAuthHeaderTest).toMatchSnapshot();

  });


});
