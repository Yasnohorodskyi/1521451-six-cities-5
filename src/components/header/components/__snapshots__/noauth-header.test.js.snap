// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render NotAuthHeader Render NotAuthHeader without store 1`] = `
<div
  className="header__nav-link header__nav-link--profile"
>
  <div
    className="header__avatar-wrapper user__avatar-wrapper"
  />
  <span
    className="header__user-name user__name"
  >
    <a
      href="/login"
      onClick={[Function]}
    >
       Sign In 
    </a>
  </span>
</div>
`;
